src/billsdepositsdialog.cpp

```diff
@@ -1348,14 +1348,18 @@ void mmBDDialog::OnsetPrevOrNextRepeatDate(wxCommandEvent& event)
     switch (repeatType)
     {
     case INXDAYS:
+        [[fallthrough]];
```

Comment:
for me VS 2022
error C2429: attribute 'fallthrough' requires compiler flag '/std:c++17'

Reply:
You are right and it needs to be C++11 compatible.